README.md

```diff
@@ -159,9 +161,11 @@ sudo apt install --yes --quiet --quiet \
 ```
 
 #### For openSUSE Tumbleweed
+`python-hwdata` is not packaged for openSUSE but can be acquired on [OBS](https://build.opensuse.org/package/show/home:tujhen:tuxclocker/python-hwdata).
```

Comment:
```suggestion
`python-hwdata` 2.4.1 is not packaged for openSUSE but can be acquired on [OBS](https://build.opensuse.org/package/show/home:tujhen:tuxclocker/python-hwdata).
```
It is packaged from what I can see, but 2.4.1 is preferred

Reply:
Don't understand in some moments, but as I explain upper, I package python-hwdata package with latest version that provides 3 packages for python 3.9, 3.10, 3.11

Reply:
Not in official (main repo), can't find with zypper, only by opi older in semi-main and latest in home:repos (OBS)

https://software.opensuse.org/package/python-hwdata?search_term=python-hwdata